### PR TITLE
Support JiraTicket multi attachments

### PR DESCRIPTION
**Pull request by asafchen-dig**

### CODE REVIEW

**Comment by asafchen-dig** [comment]
Fixes #670

File: src/components/Tests/TestTicket/index.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const TestTicket = (props: TestTicketProps) => {
       description={{
         content: renderDescription()
       }}
-      attachment={attachment}
+      attachments={[attachment]?.filter(Boolean).map((a) => a!)}
```
Comment (opoliarush) [inline src/components/Tests/TestTicket/index.tsx:67]: u can extract type for attachment and use this construction 
`fiter((item): item is Attachment => !!item)`

**Review by opoliarush — CHANGES_REQUESTED** [review]

File: src/components/Tests/TestTicket/index.tsx [hunk 2]
```diff
@@ -64,7 +65,7 @@ export const TestTicket = (props: TestTicketProps) => {
       description={{
         content: renderDescription()
       }}
-      attachments={[attachment]?.filter(Boolean).map((a) => a!)}
+      attachments={[attachment]?.filter((item): item is Attachment => !!item)}
```
Comment (kshmidt-digma) [inline src/components/Tests/TestTicket/index.tsx:68]: You can conditionally add the attachment to the array

```
  const attachments: Attachment[] = [
  ...(traceId
    ? [{
        url: `${config.jaegerURL}/api/traces/${traceId}?prettyPrint=true`,
        fileName: `trace-${traceId}.json`
      }]
    : [])
   ];
```
and avoid filtering

```
  attachments={attachments}
```

File: src/components/Insights/InsightJiraTicket/types.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ export interface InsightJiraTicketProps {
     isLoading?: boolean;
     errorMessage?: string;
   };
-  attachment?: { url: string; fileName: string };
+  attachments?: ({ url: string; fileName: string } | undefined)[];
```
Comment (kshmidt-digma) [inline src/components/Insights/InsightJiraTicket/types.ts:12]:
```suggestion
  attachments: Attachment[];
```

File: src/components/common/JiraTicket/Field/SingleField/index.tsx [hunk 4]
```diff
@@ -29,7 +29,7 @@ export const Field = (props: FieldProps) => {
 
   return (
     <s.Container $selectable={props.selectable}>
-      <s.Label>{props.label}</s.Label>
+      {props.label ? <s.Label>{props.label}</s.Label> : <></>}
```
Comment (kshmidt-digma) [inline src/components/common/JiraTicket/Field/SingleField/index.tsx:32]:
```suggestion
      {props.label && <s.Label>{props.label}</s.Label>}
```

File: src/components/common/JiraTicket/index.tsx [hunk 5]
```diff
@@ -157,26 +160,52 @@ export const JiraTicket = (props: JiraTicketProps) => {
           />
         }
       />
-      {props.attachment && (
-        <Field
-          key={"attachments"}
-          label={"Attachments"}
+      {props.attachments && props.attachments.length == 1 && (
```
Comment (kshmidt-digma) [inline src/components/common/JiraTicket/index.tsx:163]:
```suggestion
      {props.attachments && props.attachments.length === 1 && (
```

File: src/components/common/JiraTicket/types.ts [hunk 6]
```diff
@@ -17,7 +17,7 @@ export interface JiraTicketProps {
     isLoading?: boolean;
     errorMessage?: string;
   };
-  attachment?: { url: string; fileName: string };
+  attachments?: Attachment[];
```
Comment (kshmidt-digma) [inline src/components/common/JiraTicket/types.ts:20]:
```suggestion
  attachments: Attachment[];
```

File: src/components/common/JiraTicket/index.tsx [hunk 7]
```diff
@@ -157,26 +160,52 @@ export const JiraTicket = (props: JiraTicketProps) => {
...
+              text={props.attachments[0].fileName}
             />
           }
           button={
             <IconButton
               icon={DownloadIcon}
               title={"Download"}
-              onClick={handleDownloadButtonClick}
+              size={16}
+              onClick={() => handleDownloadButtonClick(props.attachments![0])}
```
Comment (kshmidt-digma) [inline src/components/common/JiraTicket/index.tsx:178]:
```suggestion
              onClick={() => handleDownloadButtonClick(props.attachments[0])}
```

File: src/components/Tests/TestTicket/index.tsx (hunk 2 above)
Comment (asafchen-dig) [reply src/components/Tests/TestTicket/index.tsx:68]: done

File: src/components/Insights/InsightJiraTicket/types.ts (hunk 3 above)
Comment (asafchen-dig) [reply src/components/Insights/InsightJiraTicket/types.ts:12]: done

File: src/components/common/JiraTicket/Field/SingleField/index.tsx (hunk 4 above)
Comment (asafchen-dig) [reply src/components/common/JiraTicket/Field/SingleField/index.tsx]: this condition is no longer there

File: src/components/common/JiraTicket/index.tsx (hunk 5 above)
Comment (asafchen-dig) [reply src/components/common/JiraTicket/index.tsx]: this condition is no longer there

File: src/components/common/JiraTicket/types.ts (hunk 6 above)
Comment (asafchen-dig) [reply src/components/common/JiraTicket/types.ts:20]: done

File: src/components/common/JiraTicket/index.tsx (hunk 7 above)
Comment (asafchen-dig) [reply src/components/common/JiraTicket/index.tsx]: this code no longer there

**Pull request merged**